### PR TITLE
#15 - Logger Intergration 

### LINKED ISSUE #15: Create a logger to log info, errors and warnings
##  Description:

Create a logger to log all the user, safe related events (info, errors, and warnings)

### PR DESCRIPTION
**Pull request by yathishram**
### Description 

This PR adds logging functionality to log errors and info using `winston`. 

### Added

Added `logger` file which wraps both `console` and logs into a single function

### CODE REVIEW

File: src/utils/logger/logger.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { success, error, warning } from '../message';
+
+
+export const logger = createLogger({
+    format: combine(
+      errors({ stack: true }),
+      json()
+    ),
+    transports: [
+      new transports.File({filename: 'info.log', level: 'info'}),
```
Comment (koshikraj) [inline src/utils/logger/logger.ts]: store all the log files inside a `logs` directory. You can probably use `path.join`

File: src/utils/safient/index.ts [hunk 2]
```diff
@@ -110,11 +109,10 @@ export class Safient {
     const safe = await safeService.get(safeId)
 
     if (safe.hasError()) {
-      console.log(error('Error while fetching safe:'), safe.getErrorMessage())
+      errorLog(`Error while fetching safe:  ${safe.getErrorMessage()}`, `Error while fetching safe ${JSON.stringify(safe.error)} @ ${Date.now()}`)
       return false
     }
-
-    console.log(success('Safe details 🔐 : '), safe.data)
+    infoLog(`Safe details 🔐 : ${safe.data}`, `'Safe Recovered' @ ${Date.now()}`)
```
Comment (koshikraj) [inline src/utils/safient/index.ts]: This is not a recovery. Remove the `'Safe Recovered'` log part

File: src/utils/logger/logger.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import {createLogger, format, transports }from 'winston';
+const { combine, errors, json } = format;
+import path from 'path'
+import { success, error, warning } from '../message';
+
+
+export const logger = createLogger({
```
Comment (koshikraj) [inline src/utils/logger/logger.ts]: Even the error logs are written to `info.log` and `warn.log`. Check this

Comment (yathishram) [reply src/utils/logger/logger.ts]: Checked this. It's working fine. All the logs are written in different files.

**Review by koshikraj — COMMENTED** [review]
Increment the package version.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**